training/common/Makefile.common

```diff
@@ -68,6 +75,28 @@ growfs:
 	# Add growfs service
 	mkdir -p build; cp -R ../common/usr build
 
+.PHONY: bootc-image-builder
+bootc-image-builder:
+	mkdir -p build/store
+	podman run \
+	  --rm \
+	  -ti \
+	  -v $(GRAPH_ROOT):/var/lib/containers/storage \
+	  $(ARCH:%=--arch %) \
```

Comment:
Does this work with the ARCH removed above?

Reply:
Ok, removing then ... I copied most of it from the Makefile.common target that is present in the /recipes section
https://github.com/containers/ai-lab-recipes/blob/d3ba2b62fbf46432fb017e2d777bec53df318fd2/recipes/common/Makefile.common#L108-L130

Reply:
Need to fix this there also.